crates/valence_new/src/client.rs

```diff
@@ -445,12 +446,23 @@ impl Client {
     ///   declining the pack (this is enforced client-side)
     /// * `prompt_message` - A message to be displayed with the resource pack
     ///   dialog.
-    pub fn set_resource_pack(
+    #[inline]
+    pub fn set_resource_pack<'a>(
+        &'a mut self,
+        url: &'a str,
+        hash: &'a str,
+        forced: bool,
+        prompt_message: Option<impl Into<Cow<'a, Text>>>,
+    ) {
+        self._set_resource_pack(url, hash, forced, prompt_message.map(|t| t.into()))
+    }
+
+    fn _set_resource_pack(
```

Comment:
Keep as one function.

Reply:
This would decrease binary size

Reply:
That does not make it any less of an unrelated change. Binary size is not a priority for us, and this PR is about fixing one issue, nothing else.